javascript/builtins/Map.json

```diff
@@ -473,6 +473,60 @@
             }
           }
         },
+        "groupBy": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Map/groupBy",
+            "spec_url": "https://tc39.es/proposal-array-grouping/#sec-map.groupby",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "preview",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.array_grouping",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": [
+                {
+                  "version_added": "preview"
+                },
+                {
+                  "version_added": "16.4",
+                  "version_removed": "preview",
+                  "alternative_name": "Array.prototype.groupToMap"
```

Comment:
After-merge review: `alternative_name` isn't really intended to be used this way.  These should be replaced with a note and `partial_implementation`.